.gitignore

```diff
@@ -1,23 +1,14 @@
-# See https://help.github.com/ignore-files/ for more about ignoring files.
-
 # Node / NPM / Yarn
 node_modules
 
 # Build
-/build
-.etherlime-store
-/localDeploy
-/run
-/vendor
 /artifacts
 /cache
 /typechain
 /abi
 
 # Secrets
-.secret
 .env
-.env.*
```

Comment:
I would keep that in (I have some other `.env.*` files beside `.env.example`, and make `.env.example` as exception.

But if others are happy with this change, I'll just update my `.git/info/exclude`

Reply:
I agree we should keep this in.

Reply:
done

Reply:
OK for me.